index.html

```diff
@@ -10,17 +10,22 @@
 </head>
 
 <body>
-    <header>
+    <header class="header-container">
         <div class="container-logo">
             <a class="logo" href="#">Abdo</a>
         </div>
         <div class="menu">
-            <a href="#" ><img src="././img/icon-Menu.svg" alt="humberger-image"></a>
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
         </div>
         <nav class="navigation">
-            <a class="links" href="#portfolio">Portfolio</a>
-            <a class="links" href="#about">About</a>
-            <a class="links" href="#contact">Contact</a>
+            <ul>
```

Comment:
- [ ] Kindly make sure that the hamburger button is visible so that it can be used to toggle the mobile navigation.
![Screen Shot 2022-09-12 at 20 08 18](https://user-images.githubusercontent.com/28307960/189725439-53e69caa-8b72-41e3-afe7-4aeeeaf9c4fa.png)